dimod/utilities.py

```diff
@@ -449,6 +450,28 @@ def child_structure_dfs(sampler, seen=None):
 
     raise ValueError("no structured sampler found")
 
+def inner_child_properties(sampler):
+    """
+    Returns the properties of inner-most child sampler in a composite.
+
+    Args:
+        sampler: A dimod sampler
+
+    Returns:
+        properties (dict): The properties of the inner-most sampler
+
+    Example:
+    >>> sampler = EmbeddingComposite(ScaleComposite(qpu_sampler))
```

Comment:
Having `EmbeddingComposite` means a dependency on dwave-system, so you'd need to skip the doctest

Reply:
is that a problem if it's in the example docstring? It just seems like the most obvious usage to me.

I wasn't aware that docstrings get tested, does that mean that they should include imports and be well self-contained?